lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -267,46 +260,27 @@ void BorrowingOperand::visitUserResultConsumingUses(
   }
 }
 
-bool BorrowingOperand::getImplicitUses(
+void BorrowingOperand::getImplicitUses(
     SmallVectorImpl<Operand *> &foundUses,
     std::function<void(Operand *)> *errorFunction) const {
   if (!areAnyUserResultsBorrowIntroducers()) {
     visitEndScopeInstructions([&](Operand *op) { foundUses.push_back(op); });
-    return false;
+    return;
   }
 
   // Ok, we have an instruction that introduces a new borrow scope and its
   // result is that borrow scope. In such a case, we need to not just add the
-  // end scope instructions of this scoped operand, but also look through any
-  // guaranteed phis and add their end_borrow to this list as well.
+  // end scope instructions of this scoped operand.
   SmallVector<BorrowingOperand, 8> worklist;
   SmallPtrSet<Operand *, 8> visitedValue;
   worklist.push_back(*this);
   visitedValue.insert(op);
-  bool foundError = false;
+
   while (!worklist.empty()) {
     auto scopedOperand = worklist.pop_back_val();
     scopedOperand.visitConsumingUsesOfBorrowIntroducingUserResults(
```

Comment:
I think this at this point should just be visitEndScopeInstructions, no?

Reply:
yes, changed it : )